python/tstUtilities.py

```diff
@@ -66,15 +66,34 @@ def getObjectFromName(objectList, name):
     return None
 
 
+def getTypeDisplayName (type):
+    """
+    In some cases the "display_name" is a generated temporary or tag-name
+    this happens for anon structures in C for example
+    
+    typedef struct {
+        int a;
+        } myType;
+
+    In that case, use use the typemark vs display_name
+    """
+
+    # The anon names are always __T<address> so use regex
+    if re.search("^__T[0-9]+$", type.display_name):
```

Comment:
I'd just use `re.match` here without `^` and `$`.

Doing an `re.search` with a hard-coded `^` and `$` on a single-line string is the same as doing `re.match`.